src/Setup/WixBA/WixBA.csproj

```diff
@@ -31,36 +60,41 @@
     </Compile>
     <Compile Include="RootViewModel.cs" />
     <Compile Include="UpdateViewModel.cs" />
+    <Compile Include="WindowProperties.cs" />
     <Compile Include="WixBA.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <None Include="WixBA.BootstrapperCore.config">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <SubType>Designer</SubType>
     </None>
   </ItemGroup>
-
-  <ItemGroup>
-    <Resource Include="Resources\exit.png" />
-    <Resource Include="Resources\gear.png" />
-    <Resource Include="Resources\legal.png" />
-    <Resource Include="Resources\news.png" />
-    <Resource Include="Resources\wrench.png" />
-  </ItemGroup>
-
   <ItemGroup>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="System" />
     <Reference Include="System.ComponentModel.DataAnnotations" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Drawing">
+      <Private>False</Private>
+    </Reference>
     <Reference Include="System.ServiceModel" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Windows.Interactivity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
If we still need this, we might need to get it from a Nuget package.  I'm not sure if this comes with VS2010.